Cargo.toml

```diff
@@ -46,25 +52,24 @@ rusqlite = "0.6.0"
 serde = "0.7.0"
 serde_json = "0.7.0"
 serde_macros = "0.7.2"
-staticfile = "0.1.0"
+staticfile = "0.2.0"
 transformable_channels = "^0.1"
 unicase = "1.3.0"
-tls = { path = "./components/tls" }
 time = "0.1"
 timer = "0.1.6"
 uuid = "0.1.18"
 url = "0.5.7"
-ws = { git = "https://github.com/housleyjk/ws-rs.git", rev = "d154fc5" }
+ws = { git = "https://github.com/JohanLorenzo/ws-rs.git" , branch = "0.4.6" }
```

Comment:
Needed, otherwise mio version mismatch. 
* https://crates.io/crates/ws/0.4.6 uses mio 0.5.0 
* whereas we use 0.5.0, but with some revisions forward https://github.com/fxbox/foxbox/pull/411/files#diff-80398c5faae3c069e4e6aa2ed11b28c0R42